modules/standard/AutoMath.chpl

```diff
@@ -1142,7 +1142,7 @@ module AutoMath {
      fewer conditionals will be evaluated at run time.
   */
   pragma "last resort"
-  @deprecated(notes="In an upcoming release 'divceil' will no longer be included by default, please 'use' or 'import' the :mod:`Math` module to call it.  Additionally its argument names will also change with this move")
+  @deprecated(notes="In an upcoming release 'divceil' will no longer be included by default, please 'use' or 'import' the :mod:`Math` module to call it.  Additionally its name and argument names will also change with this move")
```

Comment:
@lydia-duncan : In putting together https://github.com/chapel-lang/chapel/pull/23146, I found this family of error messages confusing (even as a pretty savvy user) because the tests in question already had a `use Math;` so it didn't sound like I should need to change anything.  Because the message refers to "an upcoming release" before saying "will also change", it suggested to me that the name hadn't changed yet / that I didn't need to change the name in addition to adding `use Math;`.

To me, it would read more clearly if the wording were more like "In an upcoming release, 'divceil' will no longer be available by default.  Please 'use' or 'import' the :mod:`Math` module to prepare for this move, noting that its name has changed to 'divCeil()'."

or possibly ", noting that its signature has changed to 'divCeil([new argument names and types here])`"

Reply:
I agree it won't necessarily already be, but in cases like this where it is, it was pretty confusing.  Like "You're telling me to do this, but I've already done it, so I'm skipping that sentence.  And now you're telling me it's going to change names at some point, but it's still complaining about the symbol today."  My first instinct was that we had a bug in  our deprecation logic since the code had already done what seemed to be suggested.